tests/unit/test_rate_limiting.py

```
@@ -0,0 +1,98 @@
+    limiter.enabled = True
+
+    # Re-initialize app with the new config to ensure handlers are registered
+    # because app.py disables it by default in testing mode
+    limiter.init_app(app)
+
+    # Reset storage
+    try:
+        if hasattr(limiter, "_storage") and limiter._storage:
+            limiter._storage.reset()
+    except Exception:
+        pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)